eng/Version.Details.xml

```diff
@@ -1,9 +1,5 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Private.CoreFx.NETCoreApp" Version="5.0.0-alpha.1.19563.6">
```

Comment:
There are other places that have this references, e.g.: https://github.com/dotnet/runtime/blob/master/src/libraries/pkg/Microsoft.Windows.Compatibility/tests/Directory.Build.props#L9

Delete them too?

Reply:
I think that one might be required, but I'm not sure.

@ericstj any ideas if we still need the reference to Microsoft.Private.CoreFx.NETCoreApp in the Windows Compatibility Pack?

Reply:
The Microsoft.Windows.Compatibility pack tests are no longer used, we can delete those. They were used for the old infrastructure of producing APICatalog/Port ref assembly sets.

Reply:
I'll go remove them then!